www/source/stylesheets/_pricing.scss

```diff
@@ -0,0 +1,151 @@
+.pricing-row-wrapper {
+  position: relative;
+
+  @include breakpoint(medium) {
+    .background-waves-blue {
+      position: absolute;
+      background: url('/images/background-dark-blue.png') center top repeat-x;
```

Comment:
The background image needs update, download new asset here: https://drive.google.com/open?id=0B6wx-y4RiRvoTlk5aGpXeDI3Mnc